src/routes/users/sign-up-route.js

```diff
@@ -7,31 +9,36 @@ const saveUser = user => {
 };
 
 const signUpRoute = (request, response) => {
-  // Взять данные что пришли
+  if (request.method === "POST") {
+    let body = "";
+    console.log(request);
+    request.on("data", function(data) {
+      body += data;
+
+      console.log("Incoming data!!!!");
+      console.log(body);
+    });
 
-  if (request.method === 'POST') {
-    let body = '';
+    request.on("end", function() {
+      let userData = JSON.parse(body);
+      const { username } = userData;
```

Comment:
also you should validate data that you've received from user. Now user can send you anything he wants and you still handle this request. Accepting data in specific format requirement is not met